utils/contentful-posts.ts

```diff
@@ -26,12 +26,13 @@ const transformContentfulPosts = (items: object[] = []): IBlogPost[] => {
   const posts = items.map((item: any): IBlogPost => {
     return {
       id: item.sys.id,
-      title: item.fields.title,
-      slug: item.fields.slug,
+      author: item.fields.author,
+      body: formatBody(item.fields.body) || '',
       category: item.fields.category,
-      featuredImageUrl: `https:${item.fields.featuredImage?.fields.file.url}` || '',
       datePublished: formatDate(item.sys.createdAt),
-      author: item.fields.author,
+      featuredImageUrl: `https:${item.fields.featuredImage?.fields.file.url}` || '',
```

Comment:
Similarly to my comment above, maybe this should be an optional field.

Could we use a default image (placeholder/dummy) when there's no image URL from Contentful?